piezo_web_app/PiezoWebApp/tests/services/spark_job/spark_job_service_tidy_jobs_test.py

```diff
@@ -0,0 +1,118 @@
+from unittest.mock import patch
+
+from PiezoWebApp.src.services.spark_job.spark_job_constants import CRD_GROUP
+from PiezoWebApp.src.services.spark_job.spark_job_constants import CRD_PLURAL
+from PiezoWebApp.src.services.spark_job.spark_job_constants import CRD_VERSION
+from PiezoWebApp.src.services.spark_job.spark_job_constants import NAMESPACE
+
+from PiezoWebApp.tests.services.spark_job.spark_job_service_test import TestSparkJobService
+
+
+class SparkJobServiceGetJobStatusTest(TestSparkJobService):
+    @patch('PiezoWebApp.src.services.spark_job.spark_job_service.SparkJobService.get_jobs',
+           return_value={"spark_jobs": {"job1": "RUNNING",
+                                        "job2": "PENDING",
+                                        "job3": "SUCCEEDED",
+                                        "job4": "COMPLETED",
+                                        "job5": "CrashLoopBackOff",
+                                        "job6": "UNKNOWN",
+                                        "job7": "FAILED",
+                                        "job8": "COMPLETED"},
+                         "status": 200})
```

Comment:
I'd prefer it if you mocked the `kubernetes_adapter`, rather than patching a service method.

Also, this return value is larger than I'd expect for a unit test: please (also) show how the method behaves for a single job in each of the statuses.